modules/solid_mechanics/include/materials/CombinedPlasticityStressUpdate.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "MaterialProperty.h"
+#include "BackstressRadialReturnStressUpdate.h"
+
+template <bool is_ad>
+class CombinedPlasticityStressUpdateTempl : public BackstressRadialReturnStressUpdateTempl<is_ad>
```

Comment:
CombinedPlasticityStressUpdate is too general.  I would name it something like isotropicKinematicStressUpdate.  We can wait for Ben to comment on this.  @bwspenc

Reply:
I agree that this name is too general, and we might have other models that do combined hardening. Its name should convey something about the nature of the hardening models. Something like `CombinedLinearHardening`, for instance, but this model is not just linear.